Watchman.DomainModel/Messages/Queries/GetMessagesQuery.cs

```diff
@@ -5,10 +5,19 @@ namespace Watchman.DomainModel.Messages.Queries
     public class GetMessagesQuery : PaginationMessagesQuery, IQuery<GetMessagesQueryResult>
     {
         public ulong ServerId { get; private set; }
+        public ulong ChannelId { get; private set; }
+        public ulong? UserId { get; private set; }
 
         public GetMessagesQuery(ulong serverId)
         {
             ServerId = serverId;
         }
+
+        public GetMessagesQuery(ulong serverId, ulong channelId, ulong? userId)
```

Comment:
tylko teraz pytanie czy przy `-stats month` to sie nie wysypie 
zrobiłbym to z domyślnymi wartościami 0 dla channelId i userId 
user id nie nullowalne tylko 0